mkimage.sh

```diff
@@ -139,7 +140,7 @@ fi
             --installroot="$rootfsDir" \
             --releasever="$releasever" \
             --setopt=install_weak_deps=False \
-            --nodocs --assumeyes --quiet \
+            --nodocs --assumeyes ${quiet:--quiet} \
```

Comment:
For me because of this dnf fails with this error:

`dnf: error: argument -q/--quiet: ignored explicit argument 'uiet'
`

To fix it I changed `--quiet` for` \--quiet`

Reply:
I think I've taken care of this now.